src/js/api/constants.js

```diff
@@ -1,2 +1,2 @@
-export const apiUrl = new URL("https://nf-api.onrender.com/api/v1");
-export const apiPath = apiUrl.toString()
\ No newline at end of file
+export const apiUrl = new URL('https://api.noroff.dev');
```

Comment:
While this is a valid correction, it should be reported as an issue instead of fixed directly.